tests/apollo/test_skvbc_view_change.py

```diff
@@ -359,11 +360,136 @@ async def _wait_for_read_your_writes_success(self, tracker):
                     else:
                         break
 
+    @with_trio
+    @with_bft_network(start_replica_cmd,
+                      selected_configs=lambda n, f, c: f >= 2)
+    @with_constant_load
+    async def test_f_staggered_replicas_requesting_vc(self, bft_network, skvbc, nursery):
+        """
+        The goal of this test is to check that if a subset not big enough to
+        reach quorum for execution accept CLient requests
+
+        1) Given a BFT network, we make sure all nodes are up.
+        2) crash f replicas, including the current primary.
```

Comment:
I thought the idea was to start with _f_ early replicas, and then start all the others?